doc/docportal/settings/keymaps.rst

```diff
@@ -134,45 +134,15 @@ Increase the scale factor
 Decrease the scale factor
 	*keymap_sdl-graphics_SCL-*
 
-.. _FLT1:
+.. _FLTN:
 
-Switch to nearest neighbour scaling
-	*keymap_sdl-graphics_FLT1*
+Switch to next graphical scale filter
+	*keymap_sdl-graphics_FLTN*
 
-.. _FLT2:
+.. _FLTP:
 
-Switch to AdvMame 2x/3x scaling
```

Comment:
These keymaps are also referenced in the advanced_topics/configuration_file, so that section will need to be removed and replaced with the two new refs

Reply:
Done in the new commit.